proto/cosmos/authz/v1beta1/query.proto

```diff
@@ -16,9 +16,14 @@ service Query {
     option (google.api.http).get = "/cosmos/authz/v1beta1/grants";
   }
 
-  // GranterGrants returns list of `Authorization`, granted by granter.
+  // GranterGrants returns list of `GrantAuthorization`, granted by granter.
   rpc GranterGrants(QueryGranterGrantsRequest) returns (QueryGranterGrantsResponse) {
-    option (google.api.http).get = "/cosmos/authz/v1beta1/grants/{granter}";
+    option (google.api.http).get = "/cosmos/authz/v1beta1/grants/granter/{granter}";
```

Comment:
This is client-breaking. I'm not totally sure it's worth it. Can we keep the old endpoint, or will that create some kind of clashes with the new endpoint?